gym/f110_gym/envs/observation.py

```diff
@@ -17,7 +17,7 @@ class Observation:
     :param kwargs: Additional arguments.
     """
 
-    def __init__(self, env: "F110Env"):
```

Comment:
@luigiberducci is this needed?

Reply:
Do you mean passing the `env` to the observation object? 
If so, yes because all the `observe` methods access the simulator through this handle.

Reply:
I meant the type hinting with strings, linting CI throws a fit so @nandantumu  deleted the hinting.